Makefile

```diff
@@ -110,10 +105,10 @@ publish-image-proxy: image-proxy
 	docker push ${RP_IMAGE_ACR}.azurecr.io/proxy:latest
 
 proxy:
-	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(COMMIT)" ./hack/proxy
+	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./hack/proxy
 
 run-portal:
-	go run -tags aro -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(COMMIT)" ./cmd/aro portal
+	go run -tags aro -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro portal
```

Comment:
Are we ok to have tag name as a `GitCommit` in go code?

cc @hawkowl for this. This might have an impact on what you are trying to do with ARO operator ACR env var [here](https://github.com/Azure/ARO-RP/pull/1845#discussion_r806681619).

Reply:
it appears like if there is no tag, the existing commit will be used instead. As long as it's consistently passed through, I don't think there'll be an effective change for development.

Reply:
this is exactly how it works! :)